packages/next/README.md

```diff
@@ -1761,23 +1761,7 @@ next build
 next start
 ```
 
-For example, to deploy with [`now`](https://zeit.co/now) a `package.json` like follows is recommended:
-
-```json
-{
-  "name": "my-app",
-  "dependencies": {
-    "next": "latest"
-  },
-  "scripts": {
-    "dev": "next",
-    "build": "next build",
-    "start": "next start"
-  }
-}
-```
-
-Then run `now` and enjoy!
+To deploy with [ZEIT Now](https://zeit.co/now), see the [Serverless deployment section](#serverless-deployment).
```

Comment:
This should link directly to the guide instead of redirecting to another part of the docs that then links to the guide.